packages/web/src/components/static-image/StaticImage.tsx

```diff
@@ -0,0 +1,91 @@
+import { ComponentPropsWithoutRef } from 'react'
+
+import { SquareSizes } from '@audius/common/models'
+import { Box } from '@audius/harmony'
+import {
+  DiscoveryNodeSelector,
+  productionConfig,
+  stagingConfig,
+  developmentConfig,
+  StorageNodeSelector,
+  AppAuth
+} from '@audius/sdk'
+
+import { env } from 'services/env'
+
+export type StaticImageProps = {
+  // Image CID
+  cid?: string | null
+  // Size of image to select
+  size?: SquareSizes
+  // Set height and width of wrapper container to 100%
+  fullWidth?: boolean
+  // Classes to apply to the wrapper
+  wrapperClassName?: string
+} & ComponentPropsWithoutRef<'img'>
+
+const sdkConfigs = {
+  production: productionConfig,
+  staging: stagingConfig,
+  development: developmentConfig
+}
+
+const discoveryNodeSelector = new DiscoveryNodeSelector({
+  bootstrapServices: (
+    sdkConfigs[env.ENVIRONMENT as keyof typeof sdkConfigs] ?? productionConfig
+  ).discoveryNodes
+})
+
+const auth = new AppAuth('', '')
+
+const storageNodeSelector = new StorageNodeSelector({
```

Comment:
is there a world where we can pull StorageNodeSelector from the existing sdk? iirc it's in the ssr context/app context right?

Reply:
good though, yea tried to do that before, but it causes issues bc of the async init of the sdk on the client

Reply:
word makes sense, maybe we can init the storageNodeSelector synchronous and then pass that into sdk so we have one instance? but not a big deal rn

Reply:
it looks like the discoveryNodeSelector has to wait for remote config to get the blocklist, so that would block us from making this synchronous i think?